src/styles/utils/spacing.ts

```diff
@@ -725,11 +729,19 @@ export default {
         columnGap: 12,
     },
 
+    minHeight3: {
```

Comment:
Where is this being used and why?

Reply:
I'm removing it, it's a legacy from the early part of layout coding